src/component/calculator.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+/* eslint-disable react/prop-types */
+const Buttonfunc = ({ number }) => (
+  <button type="button">{`${number}`}</button>
+);
+
+const ButtonStyle = ({ number }) => (
+  <button type="button" className="style-color">{`${number}`}</button>
+);
+
+class Calculator extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = 0;
+  }
+
+  render() {
+    return (
+      <div className="calc-grid">
+        <div className="show">
+          <h1 className="lower-show">{ this.state }</h1>
```

Comment:
- [ ] Let's change this title tag to be another `h*` as this will probably don't be the main title of our app or page. I would even consider not using an `h` tag here, maybe just a `span` or a `p` tag. Even when working with React we must ensure that the semantics are adequate on our code.